airflow_dags/dag.py

```diff
@@ -6,6 +6,7 @@
 from ip_to_country.dolt_load import get_dolt_datasets as get_ip_loaders
 from wikipedia.word_frequency.dolt_load import get_wikipedia_loaders
 from wikipedia.ngrams.dolt_load import get_dolt_datasets as get_ngram_loaders
+from five_thirty_eight.polls import get_loaders as get_five_thirty_eight_polls_loaders
```

Comment:
Not sure how much it matters, but should this file be named `dolt_load` like the others instead of `polls`?

Reply:
I plan to put a bunch of other FiveThirtyEight datasets in there.